src/build.mk

```diff
@@ -0,0 +1,18 @@
+# Standard behavior must be included here
+INCLUDE_DIRS += $(SOURCE_PATH)$(USRSRC) # add user sources to include path
+CPPSRC += $(call target_files,$(USRSRC_SLASH),*.cpp)
+CSRC += $(call target_files,$(USRSRC_SLASH),*.c)
+ 
+APPSOURCES=$(call target_files,$(USRSRC_SLASH),*.cpp)
+APPSOURCES+=$(call target_files,$(USRSRC_SLASH),*.c)
+ 
+# Custom stuff can be added here
+
+# Include all directories in /lib/
```

Comment:
Now all directories in /lib/ and /src/ are included, as well as all .cpp files in the first level of the directories in /lib/

Reply:
This is what enables us to move all of our code into source, by creating directories to organize the different parts of our project.